logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/client/ClientScope.kt

```diff
@@ -30,4 +32,6 @@ class ClientScope(
     val needsToRegisterClient: NeedsToRegisterClientUseCase get() = NeedsToRegisterClientUseCaseImpl(clientRepository)
     val registerPushToken: RegisterTokenUseCase get() = RegisterTokenUseCaseImpl(clientRepository, notificationTokenRepository)
     val deregisterNativePushToken: DeregisterTokenUseCase get() = DeregisterTokenUseCaseImpl(clientRepository, notificationTokenRepository)
+    val mlsKeyPackageCountUseCase: MLSKeyPackageCountUseCase
```

Comment:
Should this be public?

Maybe I'm missing some context. What's the usage of this use-case for the apps? I wonder if apps will ever need this. It seems like something Kalium will use internally only.

Reply:
Yes, I used in the AR to show keyPackagesCount in the debug screen.

Reply:
Nice.

Maybe we could have a `Debug` scope or something in the future. For things that should have less discoverability by consumers of Kalium.
Especially if we have some dangerous use cases for debug purposes 😆.

Reply:
Totally agree, that's the reason I'm asking for debug variant! it's hard to remember which parts we need to remove when we release the app.